test/files/run/t6502.scala

```diff
@@ -105,11 +111,31 @@ object Test extends StoreReporterDirectTest {
     println(s"test4 res2: $res2")
```

Comment:
Agreed. But I'll ask for a free pass this time as I'm just adding to the test, rather than writing it from scratch.